xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/CMakeLists.txt

```diff
@@ -50,4 +50,10 @@ if(CORE_SYSTEM_NAME STREQUAL android)
                       RendererMediaCodecSurface.h)
 endif()
 
-core_add_library(videorenderers_hwdec)
+# we might want to build on linux systems
+# with ENABLE_VDPAU=OFF and ENABLE_VAAPI=OFF
+if(CORE_SYSTEM_NAME STREQUAL linux AND NOT VDPAU_FOUND AND NOT VAAPI_FOUND)
```

Comment:
wouldn't it be easier to just do

    if(SOURCES)
        core_add_library(...)
    endif()